tutor/src/screens/assignment-grade/ux.js

```diff
@@ -19,10 +19,22 @@ export default class AssignmentGradingUX {
     @observable isPublishingScores = false;
     @observable selectedHeadingStudentsIsGrading = [];
 
-    @UiSettings.decorate('grd.hsn') hideStudentNames = false;
-    @UiSettings.decorate('grd.alpr') alphabetizeResponses = false;
-    @UiSettings.decorate('grd.soa') showOnlyAttempted = false;
-    @UiSettings.decorate('grd.sak') showAnswerKey = false;
+    // @UiSettings.decorate('grd.hsn') hideStudentNames = false;
+    // @UiSettings.decorate('grd.alpr') alphabetizeResponses = false;
+    // @UiSettings.decorate('grd.soa') showOnlyAttempted = false;
+    // @UiSettings.decorate('grd.sak') showAnswerKey = false;
+
+    get hideStudentNames() { return (UiSettings.get('grd.hsn') || false) }
+    set hideStudentNames(value) { UiSettings.set('grd.hsn', value) }
+
+    get alphabetizeResponses() { return (UiSettings.get('grd.alpr') || false) }
+    set alphabetizeResponses(value) { UiSettings.set('grd.alpr', value) }
+
+    get showOnlyAttempted() { return (UiSettings.get('grd.soa') || false) }
+    set showOnlyAttempted(value) { UiSettings.set('grd.soa', value) }
+
+    get showAnswerKey() { return (UiSettings.get('grd.sak') || false) }
+    set showAnswerKey(value) { UiSettings.set('grd.sak', value) }
```

Comment:
This isn't ideal, but I couldn't figure out what was happening with the decorator never calling `set`, this gets it working until that can be resolved.